pyacq/viewers/imageviewer.py

```diff
@@ -0,0 +1,63 @@
+from ..core import WidgetNode
+from pyqtgraph.Qt import QtCore, QtGui
+
+import numpy as np
+import vispy
```

Comment:
Let's use pyqtgraph for now; vispy's images still need some work (I plan to start on this soon)